api/v1/server/oas/transformers/worker.go

```diff
@@ -10,6 +11,32 @@ import (
 	"github.com/hatchet-dev/hatchet/pkg/repository/prisma/dbsqlc"
 )
 
+func ToWorkerLabels(labels []*dbsqlc.ListWorkerLabelsRow) *[]gen.WorkerLabel {
+	resp := make([]gen.WorkerLabel, len(labels))
+
+	for i := range labels {
+
+		var value *string
+		if v, ok := labels[i].Value.(string); ok {
```

Comment:
Can we use `.Value.Valid` and `.Value.String` here instead of casting to string and *string?